judo-ui-react/src/main/resources/actor/src/components/widgets/AggregationInput.tsx.hbs

```diff
@@ -81,7 +81,7 @@ export const AggregationInput = ({
         />
       </ButtonBase>
       {exists(value) && onView && (
-        <IconButton disabled={disabled || editMode} onClick={onView}>
+        <IconButton disabled={disabled || editMode} onClick={onView} className="astronomer-edit">
```

Comment:
Should be the className attribute after the React element just for the consistency